apps/schema/src/components/ColumnEditModal.vue

```diff
@@ -554,6 +561,27 @@ export default {
     checkForErrors() {
       this.rowErrors = getRowErrors(this.table, this.previewData);
     },
+    handleComputedChanged() {
+      if (this.column.computed) {
+        this.requiredSelect = false;
+        delete this.column.required;
```

Comment:
yes, I think this is okay, to make sure required is false when compute not null. 

However, I think this might not be best place to do this is not here because we would need to touch multiple places?
-  I would recommend to handle this server side so that it also won't fail when users import via csv/excel (easier)
- OR we should check this when executing the required in both frontend and backend (harder?)